<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePage</class>
 <widget class="QWidget" name="ProfilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(31,31,31)
}</string>
  </property>
  <widget class="QLabel" name="ProfileHeader">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>1921</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(143, 115, 101, 1);</string>
   </property>
   <property name="text">
    <string>User's Profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="NameFrame">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>880</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background-color: rgb(41,41,41);
border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>2</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(129,129,129
);
}</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameData">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>5</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border: none;
background-color: rgba(41,41,41,0);
color: rgb(220,220,220);
}</string>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="UTodos">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>522</y>
     <width>880</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background-color: rgb(41,41,41);
border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="UTodosLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>3</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(129,129,129
);
}</string>
    </property>
    <property name="text">
     <string>Uncompleted Todos</string>
    </property>
   </widget>
   <widget class="QLabel" name="UTodosData">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>13</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(220,220,220
);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="UTargets">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>584</y>
     <width>880</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background-color: rgb(41,41,41);
border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="UTargetsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-2</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(129,129,129
);
}</string>
    </property>
    <property name="text">
     <string>Unreached Targets</string>
    </property>
   </widget>
   <widget class="QLabel" name="UTargetsData">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>13</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(220,220,220
);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="RTargets">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>646</y>
     <width>880</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background-color: rgb(41,41,41);
border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="RTargetsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>8</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(129,129,129
);
}</string>
    </property>
    <property name="text">
     <string>Reached Targets</string>
    </property>
   </widget>
   <widget class="QLabel" name="RTargetsData">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>13</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(220,220,220
);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Journals">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>708</y>
     <width>880</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background-color: rgb(41,41,41);
border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="JournalsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>8</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(129,129,129
);
}</string>
    </property>
    <property name="text">
     <string>Journals</string>
    </property>
   </widget>
   <widget class="QLabel" name="JournalsData">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>13</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(220,220,220
);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="NameEditBtn">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: none;
background-image: url(&quot;src/assets/edit.png&quot;);
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
